### DISCUSSION TITLE
Autocomplete Component Multiple mode: Ability to choose the same selected Item an unlimited number of times

### DISCUSSION THREAD

**Discussion by raleighvon** [primefaces/community#1220] [PrimeReact] [answered]
I'm using AutoComplete mulitple mode and I have a unique usecase where I need to be able to select the same item an unlimited number of times and it would appear that many times in the selectedItems array as well as display that many number of chips. It doesn't actually have to be the same object but could be a copy of the item with the same value. How can I customize it to support this?

**Comment by melloware** [comment]
Chips has an `allowDuplicate` property you could probably request something similar for AutoComplete but my guess is this would have to be done inside the component as there is nothing you can customize externally to make this happen.

**Comment by raleighvon** [comment, marked as answer]
I figured out a way to select multiples of the same item or at least make it appear as such. It involves making a copy of the selected items and then modifying the duplicate item you want to select before you set the state. This can involve adding a new key to it or changing an existing keys value in it. as long as it's a copy and there is something unique about the copy, it will allow you to add multiple of the same selection. This is probably a rare situation but for what I am creating, being able to choose multiple of the same item is a requirement.
